docs/dataset-overviews/amip-sst-sea-ice-boundary-forcing.md

```diff
@@ -3,64 +3,71 @@
 <!--- source_id_stub="PCMDI-AMIP" -->
 # AMIP sea-surface temperature and sea-ice boundary forcing
 
-**This section is a work in progress.**
-**For a first draft, see https://github.com/PCMDI/input4MIPs_CVs/pull/146**
-
 ## Key contacts
 
-- Names: Paul Durack
+- Names: Paul J. Durack
 - Emails: durack1@llnl.gov
 
 ## Summary
 
 <!--- begin-cmip7-phases-source-ids -->
 <!--- Do not edit this section, it is automatically updated when the docs are built -->
```

Comment:
@durack1 I'm guessing you missed this. Can you please move all the text you wrote out of the section boundaries or update the code which auto-generates this text

Reply:
ha ha, where is the code? This is one of those examples of keep it simple that has tripped me up, and I would consider myself a "sophisticated" user..